### PR TITLE
docs: fix TanStack & TypeScript typos

### PR DESCRIPTION
**Pull request by ruisaraiva19**

### CODE REVIEW

File: docs/framework/react/migrate-from-react-router.md [hunk 1]
```diff
@@ -10,25 +10,25 @@ Here is the [example repo](https://github.com/Benanna2019/SickFitsForEveryone/tr
...
   - [ ] Set `to` property and `params` property where needed
-- [ ] Replace any React Router `Outlet`'s with the Tanstack/router equivalent
+- [ ] Replace any React Router `Outlet`'s with the TanStack Router equivalent
 - [ ] If you are using `useSearchParams` hook from React Router, move the search params default value to the validateSearch property on a Route definition.
-  - [ ] Instead of using the `useSearchParams` hook, use Tanstack/router `Link`'s search property to update the search params state
+  - [ ] Instead of using the `useSearchParams` hook, use TanStack Router `Link`'s search property to update the search params state
   - [ ] To read search params you can do something like the following
     - `const { page } = useSearch({ from: productPage.fullPath })`
-- [ ] If using React Router’s `useParams` hook, update the import to be from Tanstack/router and set the `from` property to the literal path name where you want to read the params object from
+- [ ] If using React Router’s `useParams` hook, update the import to be from TanStack Router and set the `from` property to the literal path name where you want to read the params object from
```
Comment (schiller-manuel) [inline docs/framework/react/migrate-from-react-router.md:31]: should be the correct import package here and in the other places where a slash is used

File: docs/framework/react/migrate-from-react-router.md [hunk 2]
```diff
@@ -10,25 +10,25 @@ Here is the [example repo](https://github.com/Benanna2019/SickFitsForEveryone/tr
...
   - [ ] Add `to` prop with literal path
   - [ ] Add `params` prop, where necessary with params like so `params={{ orderId: [order.id](http://order.id) }}`
-- [ ] Replace all instances of React Router `useNavigate` hook with Tanstack/router `useNavigate` hook
+- [ ] Replace all instances of React Router `useNavigate` hook with TanStack Router `useNavigate` hook
   - [ ] Set `to` property and `params` property where needed
-- [ ] Replace any React Router `Outlet`'s with the Tanstack/router equivalent
+- [ ] Replace any React Router `Outlet`'s with the TanStack Router equivalent
 - [ ] If you are using `useSearchParams` hook from React Router, move the search params default value to the validateSearch property on a Route definition.
-  - [ ] Instead of using the `useSearchParams` hook, use Tanstack/router `Link`'s search property to update the search params state
+  - [ ] Instead of using the `useSearchParams` hook, use TanStack Router `Link`'s search property to update the search params state
```
Comment (schiller-manuel) [inline docs/framework/react/migrate-from-react-router.md:28]: should be the correct import package

**Review by schiller-manuel — CHANGES_REQUESTED** [review]

File: docs/framework/react/migrate-from-react-router.md (hunk 1 above)
Comment (ruisaraiva19) [reply docs/framework/react/migrate-from-react-router.md]: Updated ✅

File: docs/framework/react/migrate-from-react-router.md (hunk 2 above)
Comment (ruisaraiva19) [reply docs/framework/react/migrate-from-react-router.md]: Updated ✅

**Pull request merged**